src/scraper.py

```diff
@@ -0,0 +1,47 @@
+import feedparser
+
+AUTHORIZED_TOPICS = [
```

Comment:
it's more of a set of "selected" topics than authorized (we load all top news on top of it that may not be in authorized

Reply:
The topics are quite generic and fixed, not every topic is available. This list corresponds to the banner at the top of Google News
![image](https://github.com/veritas-serum/veritaserum/assets/12557936/48652e03-05df-4df8-a3b7-5bde1ce39d58)